### ISSUE TITLE
Shutdown VMs with expired grid proxy.

### ISSUE THREAD

**Issue by fransham** [hep-gc/cloud-scheduler#67]
I ran into the followin situation:  I submitted many jobs to a queue, and the cloud scheduler successfully booted VMs to run the jobs.  My proxy expired as my jobs were running.  When all of the jobs finished, the cloud scheduler tried to shutdown the VMs it had created for my jobs, but couldn't since my proxy was expired.  When I did a grid-proxy-init on the cloud sheduler's machine, it successfully destroyed the VMs.  

I'd suggest giving the cloud scheduler the power to shut down VMs regardless of whether or not the user has a valid proxy.  I can forsee a situation where a user submits a bunch of jobs, checks his results, but doesn't realize that all of his vms are still up and running, idly taking up resources from whoever else would like to use them.

In short: if a user has no more jobs in the queue, the cloud scheduler should always be able to shutdown the unneeded VMs.
